test/unary_test.js

```diff
@@ -736,4 +736,53 @@ describe('test unary', function() {
         ],
         [3, 2, 2, 1]);
   });
+
+  it('sqrt', function() {
+    testUnary(
+        'sqrt',
+        [0, 1, 4, 9],
+        [0, 1, 2, 3],
+        [4]);
+    testUnary(
+        'sqrt',
+        [
+          0,  1, 4,
+          9, 16, 25,
+        ],
+        [
+          0, 1, 2,
+          3, 4, 5,
+        ],
+        [2, 3]);
+    testUnary(
+        'sqrt',
+        [
+          0,   1,   4,
+          9,  16,  25,
+          36,  49,  64,
+          81, 100, 121,
+        ],
+        [
+          0,  1,  2,
+          3,  4,  5,
+          6,  7,  8,
+          9, 10, 11,
+        ],
+        [2, 2, 3]);
+    testUnary(
+        'sqrt',
+        [
+          0,   1,   4,
+          9,  16,  25,
+          36,  49,  64,
+          81, 100, 121,
+        ],
+        [
+          0,  1,  2,
+          3,  4,  5,
+          6,  7,  8,
+          9, 10, 11,
+        ],
```

Comment:
Should we also tests with floating point results with appropriate ULP tolerance setting?

Reply:
I just added simple float test for sqrt op. 
Since the tests of baseline are float64, it would involve more efforts to define ULP tolerance for float64 precision which current we have tolerance definitions of float32 and float16, how about testing ULP tolerance by WebNN WPT tests? Thanks.

Reply:
It makes sense. Let's keep the unit tests here simple.